src/main/java/it/gov/pagopa/payhub/activities/dto/classifications/ClassifyDTO.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.payhub.activities.dto.classifications;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class ClassifyDTO implements Serializable {
+    private Long organizationId;
+    private Long transferId;
+    private Long paymentNotifyId;
+    private Long paymentReportingId;
+    private Long treasuryId;
+    private String classificationCode;
```

Comment:
when we will have the enum, let's use it as type of this field

Reply:
done